app_spec/zomes/blog/code/src/post.rs

```diff
@@ -53,9 +56,22 @@ pub fn definition() -> ValidatingEntryType {
             hdk::ValidationPackageDefinition::ChainFull
         },
 
-        validation: |post: crate::post::Post, _ctx: hdk::ValidationData| {
-            (post.content.len() < 280)
-                .ok_or_else(|| String::from("Content too long"))
+        validation: |data: ValidationData<Post>| {
```

Comment:
As discussed in person with half of the team right now, we would like to get rid of the app specific type here (`Post`) since it's defined up as the `native_type` which is currently not used.